spark/src/main/scala/org/apache/comet/serde/QueryPlanSerde.scala

```diff
@@ -1838,6 +1840,62 @@ object QueryPlanSerde extends Logging with ShimQueryPlanSerde {
           }
         }
 
+      case join: SortMergeJoinExec if isCometOperatorEnabled(op.conf, "sort_merge_join") =>
+        // `requiredOrders` and `getKeyOrdering` are copied from Spark's SortMergeJoinExec.
+        def requiredOrders(keys: Seq[Expression]): Seq[SortOrder] = {
+          keys.map(SortOrder(_, Ascending))
+        }
+
+        def getKeyOrdering(
```

Comment:
curious why we can't use `outputOrdering` from the join

Reply:
`outputOrdering` is not exactly same as the key ordering here. The key ordering only counts in left keys ordering. But `outputOrdering`, for example, is empty for full join.